community/terra/qis_adv/quantum_walk.ipynb

```diff
@@ -0,0 +1,570 @@
+    "        cnx(qc,*qubits[:-2],qubits[-1])\n",
+    "        \n",
+    "        #B matrix (cry again, but opposite angle)\n",
+    "        qc.cu3(-np.pi/2, 0, 0, qubits[-2], qubits[-1])\n",
+    "        \n",
+    "        #Control\n",
+    "        cnx(qc,*qubits[:-2],qubits[-1])\n",
+    "        \n",
+    "        #C matrix (final rotation)\n",
+    "        qc.crz(-np.pi/2,qubits[-2],qubits[-1])\n",
+    "    elif len(qubits)==3:\n",
+    "        qc.ccx(*qubits)\n",
+    "    elif len(qubits)==2:\n",
+    "        qc.cx(*qubits)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Decide the number of qubits $n$ for represent walker's state. (Whole circuits requires $n+1$ qubits)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "lHI4G7fgT9Wn"
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from qiskit import IBMQ, QuantumCircuit, ClassicalRegister, QuantumRegister, execute\n",
+    "from qiskit.tools.visualization import plot_histogram,plot_state_city\n",
+    "\n",
+    "n=3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 85
+    },
+    "colab_type": "code",
+    "id": "kLSxVQxerGyo",
+    "outputId": "b319f1d1-b5aa-4113-e12b-eecbf993a362"
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Remote backend \"ibmqx_qasm_simulator\" could not be instantiated due to an invalid config: {'local': ['Missing data for required field.'], 'memory': ['Missing data for required field.'], 'gates': {0: {'qasm_def': ['Missing data for required field.'], 'name': ['Missing data for required field.'], 'parameters': ['Missing data for required field.']}}, 'max_shots': ['Missing data for required field.'], 'conditional': ['Missing data for required field.'], 'backend_name': ['Missing data for required field.'], 'basis_gates': ['Missing data for required field.'], 'n_qubits': ['Missing data for required field.'], 'backend_version': ['Missing data for required field.'], 'open_pulse': ['Missing data for required field.']}\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[<IBMQBackend('ibmq_qasm_simulator') from IBMQ(ibm-q-keio, keio-internal, keio-students)>,\n",
+       " <IBMQBackend('ibmq_20_tokyo') from IBMQ(ibm-q-keio, keio-internal, keio-students)>,\n",
+       " <IBMQBackend('ibmq_poughkeepsie') from IBMQ(ibm-q-keio, keio-internal, keio-students)>]"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "IBMQ.load_accounts()\n",
+    "#IBMQ.enable_account(\"53a94ea70a24c85cfac5f5e87e369d813f5efff70fed9c973b2b3637be5b2e0378da1921b5bbd786574e74d44dd8a64dda3095dbbda528b414c1b567476a87b2\")\n",
```

Comment:
Remove this comment, since I believe this is one of your IBM Q access tokens. You should regenerate this as soon as possible for security.